aws/buckets/log.tf

```diff
@@ -0,0 +1,15 @@
+#
+# Bucket access logs, stored for 30 days
+#
+module "log_bucket" {
+  source            = "github.com/cds-snc/terraform-modules//S3_log_bucket?ref=17994187b8628dc5decf74ead84768501378df4c" # ref for v10.0.0
+  bucket_name       = "cds-data-lake-bucket-logs-${var.env}"
```

Comment:
Same comment about a bucket name clash here.  I find it interesting that bucket names need to be globally unique.